### PR TITLE
 3 new topographical map images w/ fixed pine valley lake

### PR DESCRIPTION
**Pull request by illudedPerception**
*fixed the lake size and added back a few lines i forgot to move from another layer

All new topographical map images for all maps for a consistent look. With 5 meter step and labels to indicate the height, can't do SVGs or the labels will not export, the matrix transform settings were exported from QGIS with the PNG and when i tested Pine valley in a local version of Studio the settings worked perfect
matrix transform settings
Pine valley
-94.9
0
0
-94.9
199952
199952

Lake valley
-92.74
0
0
-92.74
197453
197453

Aurora falls (seems to be 6x6km map)
-141.86
0
0
-141.86
300459
300899

### CODE REVIEW

**Comment by illudedPerception** [comment]
Looks good to me now if used with QGIS export settings

**Review by scottanderson — CHANGES_REQUESTED** [review]
Could you please update `renderBackground()` in [RailroadMap.ts](https://github.com/scottanderson/railroad.studio/blob/fbe709d9b1c6af6eabe6fd58d66b07a2300ba2c6/ts/RailroadMap.ts#L744-L759) to make use of these files?

I'm not a big fan of the green background on the labels. Could that be white instead?

**Comment by illudedPerception** [comment]
> Could you please update `renderBackground()` in [RailroadMap.ts](https://github.com/scottanderson/railroad.studio/blob/fbe709d9b1c6af6eabe6fd58d66b07a2300ba2c6/ts/RailroadMap.ts#L744-L759) to make use of these files?
> 
> I'm not a big fan of the green background on the labels. Could that be white instead?
* "Could you please update `renderBackground()`" will do

* "> I'm not a big fan of the green background on the labels. Could that be white instead?" me neither but its to understand what height goes to what line maybe antoher day il deal with the labels i got an idea but its getting late today

**Review by illudedPerception — COMMENTED** [review]
should be fine

**Comment by illudedPerception** [comment]
well shit i need to do the labels now cause i forgot the matrix stuff changes a bit when i changes stuff in the map image so dont merge il change the labels

**Comment by illudedPerception** [comment]
it's good in my eyes now

File: ts/RailroadMap.ts [hunk 1]
```diff
@@ -745,13 +745,16 @@ export class RailroadMap {
...
+            transform = 'matrix(-92.74,0,0,-92.74,197453,197453)';
             // image = 'LakeValleyMap2.png';
             // transform = 'matrix(-200,0,0,-200,200000,200000)';
-        } else {
-            image = 'RRO_Pine_Valley_topo_map.png';
-            transform = 'matrix(-116.75,0,0,-116.75,233700,231900)';
+        } else if (this.railroad.settings.gameLevelName === 'AuroraFalls') {
+            image = 'AuroraFallsTopo.png';
+            transform = 'matrix(-141.86,0,0,-141.86,300459,300899)';
+        } else if ('PineValley') {
```
Comment (scottanderson) [inline ts/RailroadMap.ts]: Strings are always truthy as long as they are non-empty, so this behaves the same as:

```suggestion
        } else {
```

File: ts/RailroadMap.ts [hunk 2]
```diff
@@ -760,11 +763,25 @@ export class RailroadMap {
...
+                .translate(-2_000_00, -2_000_00)
+                .radius(100_00)
+                .addClass('map-border');
+        } else if (this.railroad.settings.gameLevelName === 'AuroraFalls') {
+            return this.layers.border
+                .rect(6_000_00, 6_000_00)
+                .translate(-3_000_00, -3_000_00)
+                .radius(100_00)
+                .addClass('map-border');
+        } else (this.railroad.settings.gameLevelName === 'PineValley'); {
```
Comment (scottanderson) [inline ts/RailroadMap.ts]: Pine valley maps have gameLevelName set to null.

The `;` makes the code behave the same as:

```suggestion
        } else {
```

File: ts/RailroadMap.ts [hunk 3]
```diff
@@ -760,11 +763,25 @@ export class RailroadMap {
...
-            .translate(-2_000_00, -2_000_00)
-            .radius(100_00)
-            .addClass('map-border');
+        if (this.railroad.settings.gameLevelName === 'LakeValley') {
+            return this.layers.border
+                .rect(4_000_00, 4_000_00)
+                .translate(-2_000_00, -2_000_00)
+                .radius(100_00)
+                .addClass('map-border');
+        } else if (this.railroad.settings.gameLevelName === 'AuroraFalls') {
```
Comment (scottanderson) [inline ts/RailroadMap.ts:772]: Can you please share an AF save file for testing?

Comment (illudedPerception) [reply ts/RailroadMap.ts:772]: here it is
[!AF_main.zip](https://github.com/scottanderson/railroad.studio/files/14379576/AF_main.zip)

File: ts/industries.ts [hunk 4]
```diff
@@ -395,12 +395,82 @@ export const industrySvgPaths: Partial<Record<IndustryName, Record<string, PathA
...
     'watertower_small': waterTower,
-    'WaterWell': waterTower,
     'engineshed_style1': largeEngineHouse,
     'engineshed_style2': largeEngineHouse,
     'engineshed_style3': largeEngineHouse,
     'engineshed_style4': largeEngineHouse,
     'Woodrick': {
         'building': rect(-130, -65, 260, 130),
     },
+    'WaterWell': {
```
Comment (illudedPerception) [inline ts/industries.ts:405]: this is only the farm industry chain, but i am ready to trace the gold industry chain as well

File: railroad.studio/style.css [hunk 5]
```diff
@@ -68,6 +68,10 @@
   --coaling-tower: #fc0;
   --building-beige: #cc9;
   --building-old: #663;
+  --fence-fill: transparent;
```
Comment (illudedPerception) [inline railroad.studio/style.css:71]: for the fences

File: railroad.studio/style.css [hunk 6]
```diff
@@ -68,6 +68,10 @@
...
+  --fence-stroke: black;
+  --walkway-stroke: black;
```
Comment (illudedPerception) [inline railroad.studio/style.css:73]: for the platforms what are not for cargo

File: ts/RailroadMap.ts (hunk 3 above)
Comment (illudedPerception) [reply ts/RailroadMap.ts:772]: it looks like this tho
image
image

**Review by scottanderson — APPROVED** [review]

**Comment by scottanderson** [comment]
🚀

**Pull request merged**